wgsl/index.bs

```diff
@@ -1985,9 +1985,8 @@ memory locations.
 Two sets of memory locations <dfn noexport>overlap</dfn> if the intersection of
 their sets of memory locations is non-empty. Each variable declaration has a
 set of memory locations that does not overlap with the sets of memory locations of
-any other variable declaration. Memory operations on structures and arrays may
-access padding between elements, but it is a [=dynamic error=] to access
-padding at the end of the structure or array.
+any other variable declaration. Memory operations on structures and arrays
+must not access padding memory locations.
```

Comment:
Should be phrased as "will not" instead of "must not".

For example, compare with 

> Two variables with overlapping lifetimes will not have [overlapping memory](https://gpuweb.github.io/gpuweb/wgsl/#overlap).

or 

> The loop body will execute five times.